feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -365,3 +388,169 @@ fun CaptureButton(
         })
     }
 }
+
+enum class ToggleState {
+    Left,
+    Right
+}
+
+@Composable
+fun ToggleButton(
+    leftIcon: Painter,
+    rightIcon: Painter,
+    modifier: Modifier = Modifier.width(64.dp).height(32.dp),
+    initialState: ToggleState = ToggleState.Left,
+    onToggleStateChanged: (newState: ToggleState) -> Unit = {},
+    isDisabled: Boolean = false,
+    iconPadding: Dp = 8.dp
+) {
+    val backgroundColor = MaterialTheme.colorScheme.surface
```

Comment:
This should be `MaterialTheme.colorScheme.surfaceContainerHighest` according to the [spec](https://m3.material.io/components/icon-buttons/specs#5309610a-4515-44f9-830d-880e2a2240a2)
![image](https://github.com/google/jetpack-camera-app/assets/1455403/e0d017f1-5345-4e14-9e5e-926969316742)

When it's disabled, it should also be `disableColor` with an alpha of `0.12`:
![image](https://github.com/google/jetpack-camera-app/assets/1455403/c43bed67-a96e-4dbe-ab38-a38d0a5275d9)

Reply:
Thank you, updated the background color to "surfaceContainerHighest". Since the disable color is already "onSurface", using 0.12 alpha for container and 0.38 for the icon, I left them there.